cmd/influx/pkg.go

```diff
@@ -333,6 +430,15 @@ func (b *cmdPkgBuilder) pkgExportAllRunEFn(cmd *cobra.Command, args []string) er
 func (b *cmdPkgBuilder) cmdPkgExportStack() *cobra.Command {
 	cmd := b.newCmd("stack $STACK_ID", b.pkgExportStackRunEFn, true)
 	cmd.Short = "Export all existing resources for an organization as a package"
+	cmd.Long = `
+	The pkg export stack command exports the resources associated with a stack as
+	they currently exist in the platform. All the same metadata.name fields will be
+	reused.
+
+	Example:
+		# Export by a stack
+		influx pkg export stack $STACK_ID
```

Comment:
v2 docs haven't been added yet. Should be available come next release. Will backfill that when it is available. Perhaps we can do it in tandem @sanderson?

Reply:
@jsteenb2 This should be pretty easy to add. I'll do it this afternoon. The task-based stack management docs are a different PR and will get finalized once I wrap up my InfluxDays presentation.

Reply:
awesome, sounds great 👍

Reply:
https://github.com/influxdata/docs-v2/pull/1083